### DISCUSSION TITLE
Code options preview crops wrapped lines at large font face

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#109350] [Mobile] [answered]
Google Pixel 3a
GitHub Mobile beta for Android

1. Open a pull request files view
2. Tap the ⚙️
3. Enable override system font size
4. Drag the slider to the right 
![Screenshot_20240220-094254.png](https://github.com/community/community/assets/2119212/bd94db95-bd9c-4b9f-9345-a6e8857e111c)

**Comment by iskae** [comment, marked as answer]
Hi @jsoref 👋 

Thank you for reaching out 😊 I was able to reproduce the issue and we will be working on a fix for this 🙇 

Please stay tuned for future releases!
